python/cuml/test/utils.py

```diff
@@ -163,3 +163,21 @@ def quality_param(*args, **kwargs):
 
 def stress_param(*args, **kwargs):
     return pytest.param(*args, **kwargs, marks=pytest.mark.stress)
+
+
+class ModuleConfig:
```

Comment:
Add a simple docstring to help devs in the future

Reply:
Agreed!